chrome-launcher/test/chrome-launcher-test.ts

```diff
@@ -102,4 +102,24 @@ describe('Launcher', () => {
     assert.strictEqual(pid, chromeInstance.pid);
     await chromeInstance.kill();
   });
+
+  it('launching flags without --disable-extensions', async () => {
+    const spawnStub = stub().returns({pid: 'some_pid'});
+
+    const chromeInstance = new Launcher(
+        {enableExtensions: true},
+        {fs: fsMock as any, rimraf: spy() as any, spawn: spawnStub as any});
+    stub(chromeInstance, 'waitUntilReady').returns(Promise.resolve());
+
+    chromeInstance.prepare();
+
+    try {
+      await chromeInstance.launch();
+    } catch (err) {
+      return Promise.reject(err);
+    }
+
+    const chromeFlags = spawnStub.getCall(0).args[1] as string[];
+    assert.ok(chromeFlags.indexOf('--disable-extensions') < 0)
```

Comment:
> To solve the error, we need to change target config in tsconfig.json from 'es6' to 'es2016'

gah. I would say switch back to `indexOf`, it's the fast fix we can land now and leave switching the target to a different PR.

(@paulirish's 👍 may mean he supports switching over to `es2016` now, so we'll let him speak up if that's the case :)